libs/platform/src/lib/components/wizard-generator/wizard-generator.service.ts

```diff
@@ -265,6 +265,16 @@ export class WizardGeneratorService {
         return this._submittedFormRawValues[stepId] !== undefined;
     }
 
+    /**
+     *
+     * @returns {Boolean} if steps are untouched, will return true, if yes - false
+     */
+    isStepsUntouched(): boolean {
+        return ![...this.stepsComponents].some(([_, component]) =>
+            component.forms.some((item) => Object.values(item.form.value).some((x) => x !== undefined && x !== ''))
```

Comment:
I think this might not work appropriately due to the ability to set a default value. Maybe check for dirty or touched instead?